src/auditlog/diff.py

```diff
@@ -135,7 +135,7 @@ def model_instance_diff(old, new):
         new_value = get_field_value(new, field)
 
         if old_value != new_value:
-            diff[field.name] = (smart_text(old_value), smart_text(new_value))
+            diff[model_fields['mapping_fields'].get(field.name, field.name)] = (smart_text(old_value), smart_text(new_value))
```

Comment:
I would rather keep this dictionary with just field names and do some mapping elsewhere (for display only). In that case the default could be the verbose name of the field (if it still exists).